rpcs3/rpcs3qt/game_list_frame.cpp

```diff
@@ -532,7 +532,33 @@ void game_list_frame::Refresh(const bool fromDrive, const bool scrollAfter)
 		add_dir(_hdd + "game/", false);
 		add_dir(_hdd + "disc/", true);
 
-		for (auto pair : YAML::Load(fs::file{fs::get_config_dir() + "/games.yml", fs::read + fs::create}.to_string()))
+		auto get_games = []() -> YAML::Node
+		{
+			try
+			{
+				fs::file games(fs::get_config_dir() + "/games.yml", fs::read + fs::create);
+
+				if (games)
+				{
+					return YAML::Load(games.to_string());
+				}
+				else
+				{
+					game_list_log.error("Failed to load games.yml, check permissions.");
+					return {};
+				}
+			}
+			catch (...)
```

Comment:
Don't they throw anything more generalized? Might be worth catching that instead of a catch-all.

Reply:
They do but literally who cares.

Reply:
You could catch some fatal error from `fs::file` constructor this way for example.